backend/src/test/java/develup/application/mission/MissionServiceTest.java

```diff
@@ -26,8 +24,9 @@ class MissionServiceTest {
     @Test
     @DisplayName("모든 미션을 조회한다.")
     void getMissions() {
-        missionRepository.save(new Mission("미션 1", Language.JAVA, "미션 설명", "미션 썸네일", "미션 url"));
-        missionRepository.save(new Mission("미션 2", Language.JAVA, "미션 설명", "미션 썸네일", "미션 url"));
+        MissionTestData.MissionBuilder missionBuilder = MissionTestData.defaultMission();
+        missionRepository.save(missionBuilder.build());
+        missionRepository.save(missionBuilder.build());
```

Comment:
같은 부분이라 build를 하고 넣어도 괜찮겠네요 👍

```suggestion
        Mission mission = MissionTestData.defaultMission().build();
        missionRepository.save(mission);
        missionRepository.save(mission);
```

Reply:
구두로 말씀 드린 것처럼 같은 엔티티를 두 번 저장할 수 없어서 분리했습니다!